### ISSUE TITLE
LINQ expression support.

### ISSUE THREAD

**Issue by miniwolf** [mono/debugger-libs#183]
I am trying to write an extension in VS Code that debugs Unity projects.
For this I am using `StackFrame.GetExpressionValue(string expression, EvaluationOptions options)`. The result that I getting back is that the expression is not supported.

The expression is based on an array that I create: 
`int[] scores = { 97, 92, 81, 60 };`
Expression I run in the Debugger console is this: `scores.Select(i => i)`

Is there something that I am supposed to do differently?

**Comment by DavidKarlas** [comment]
Are you building debugger-libs yourself for your extension? And using master version of debugger-libs?

**Comment by miniwolf** [comment]
Yes and yes. I have also tried using the ones that arrives with Visual Studio for Mac.

**Comment by miniwolf** [comment]
So any ideas about this?

**Comment by jstedfast** [comment]
@DavidKarlas wasn't this already implemented?

**Comment by DavidKarlas** [comment]
I think some scenarios are not handled yet.
